src/main/drivers/display.c

```diff
@@ -77,14 +77,28 @@ int displayWrite(displayPort_t *instance, uint8_t x, uint8_t y, const char *s)
 {
     instance->posX = x + strlen(s);
     instance->posY = y;
-    return instance->vTable->writeString(instance, x, y, s);
+    return instance->vTable->writeString(instance, x, y, s, TEXT_ATTRIBUTES_NONE);
+}
+
+int displayWriteAttr(displayPort_t *instance, uint8_t x, uint8_t y, const char *s, textAttributes_t attr)
```

Comment:
`displayWriteAttr` name is confusing - it gives an impression that function will only write attributes. Probably `displayWriteWithAttr` is a better name?

Reply:
Seems like a good idea. I've just added a commit renaming those functions.